akka-http-tests/src/main/java/akka/http/javadsl/server/examples/simple/SimpleServerHttpHttpsApp.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.http.javadsl.server.examples.simple;
+
+import akka.NotUsed;
+import akka.actor.ActorSystem;
+import akka.http.javadsl.ConnectHttp;
+import akka.http.javadsl.Http;
+import akka.http.javadsl.HttpsConnectionContext;
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.javadsl.model.HttpResponse;
+import akka.http.javadsl.server.AllDirectives;
+import akka.http.javadsl.server.Route;
+import akka.stream.ActorMaterializer;
+import akka.stream.javadsl.Flow;
+
+import java.io.IOException;
+
+public class SimpleServerHttpHttpsApp extends AllDirectives { // or import Directives.*
+
+  public Route createRoute() {
+    return get( () -> complete("Hello World!") );
+  }
+
+  // ** STARTING THE SERVER ** //
+
+  public static void main(String[] args) throws IOException {
+    final ActorSystem system = ActorSystem.create("SimpleServerHttpHttpsApp");
+    final ActorMaterializer materializer = ActorMaterializer.create(system);
+    final Http http = Http.get(system);
+
+    final SimpleServerApp app = new SimpleServerApp();
+    final Flow<HttpRequest, HttpResponse, NotUsed> flow = app.createRoute().flow(system, materializer);
+
+    //#both-https-and-http
+    Http.get(system).bindAndHandle(flow, ConnectHttp.toHost("localhost", 80), materializer);
+
+    HttpsConnectionContext https = SimpleServerApp.useHttps(system, http);
```

Comment:
The SimpleServerHttpHttpsApp code snippet appears in the doc, right next to the section explaining SimpleServerApp. So I thought we don't want to repeat the same code on the same page of the doc.

Let me add more description and comments to clarify this.